python/requirements.txt

```diff
@@ -1,7 +1,9 @@
 requests>=2.20.0
-numpy>=1.13
+numpy>=1.13 ; python_version>="3.5" and platform_system != "Windows"
+numpy>=1.13, <=1.19.3 ; python_version>="3.5" and platform_system == "Windows"
```

Comment:
Plz notice that the numpy version changes on develop branch is reverted, is this needed?

Reply:
No, it's just a copy-paste error, thank you for your awareness